libs/pika/async_mpi/include/pika/async_mpi/mpi_polling.hpp

```diff
@@ -64,40 +65,41 @@ namespace pika::mpi::experimental {
 
         const char* stream_name(stream_type s);
 
+        // -----------------------------------------------------------------
+        /// Adds an mpi request to the internal queues for polling/handling
         PIKA_EXPORT bool add_request_callback(request_callback_function_type&&, MPI_Request);
-        PIKA_EXPORT void register_polling(pika::threads::detail::thread_pool_base&);
-        PIKA_EXPORT void unregister_polling(pika::threads::detail::thread_pool_base&);
```

Comment:
This seems to have disappeared from the header. Just checking: is this because it's no longer needed elsewhere, only internally?

Reply:
correct. The user no longer explicitly creates an mpi pool, instead, during init, you call mpi::pool_create and it sets everything up, there is still a register polling function, but it moved and does not take a pool param since the mpi code knows which pool to use.